.github/workflows/tf-plan.yaml

```diff
@@ -115,20 +115,29 @@ jobs:
         env:
           TERRAFORM_PLAN_PATH: ${{ steps.plan.outputs.text_plan_path }}
         run: |
+          MAX_PLAN_LENGTH=120000
           TERRAFORM_PLAN=$(cat $TERRAFORM_PLAN_PATH)
           TERRAFORM_SUMMARY_HEADER=$(grep -E -i -m 1 'No changes|Plan:|Outputs:' <<< "$TERRAFORM_PLAN")
-
+          PLAN_LENGTH=${#TERRAFORM_PLAN}         
+          START=$((PLAN_LENGTH - MAX_PLAN_LENGTH))
+          if [ "$START" -lt 0 ]; then
+            echo "Plan will not be cut !"
```

Comment:
@Davidoutz [cosmetic] In the positive scenario, I'd prefer having no extra output (i.e. no `echo` statement)